src/main/java/com/ss/training/utopia/agent/dao/AirportDAO.java

```diff
@@ -0,0 +1,13 @@
+package com.ss.training.utopia.agent.dao;
+
+import com.ss.training.utopia.agent.entity.Airport;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * @author Trevor Huis in 't Veld
+ */
+@Repository
+public interface AirportDAO extends JpaRepository<Airport, Long> {
+    Airport findByAirportId(Integer id);
```

Comment:
I'm pretty sure this isn't needed; you can just use the built-in `findById`. If you were to keep this, you'd want to make the parameter a Long instead of an Integer.